Branded sidebar

sean-roberts opened the pull request:
work for https://github.com/hypothesis/product-backlog/issues/214

I used `help.html.jinja2` in the h directory to help me work on this. I recommend using that for testing as well.

Settings I used to get it like we proposed to eLife:

```
     "branding": {
        "highlightColor": "#1689CE",
        "appBackgroundColor": "#fff",
        "ctaBackgroundColor": "#1689CE",
        "ctaTextColor": "#fff",
        "selectionFontFamily": "georgia",
        "annotationFontFamily": "georgia"
      }
```

For reference, hat we presented to elife: https://docs.google.com/presentation/d/1XSQoSwfmuPjtPPW6mUeGo_4d1YnBB7ds1fpP939XqMg/edit#slide=id.g1c59f62e74_0_7

### What to test
Just play around with the settings and make sure they are propagating properly. There is some "massaging" of the UI that we planned on doing to make it play well with elife's design. Arti and I will go over the latter part. Code reviewer reference the ticket mentioned above and review the areas affected by settings.

sean-roberts commented:
Marking as WIP until I can sync up with Arti on where this is and some small UI tweaks

robertknight commented:
I gave this a brief test. The font and branding colour changes are visible. On pages without any/many annotations the app background colour does not span the full height of the sidebar as the container `<div>` doesn't stretch to the full height of the iframe:

<img width="478" alt="screenshot 2017-03-24 07 36 19" src="https://cloud.githubusercontent.com/assets/2458/24284739/b3602dc0-1064-11e7-8b40-f4a86219d44b.png">

> There is some "massaging" of the UI that we planned on doing to make it play well with elife's design

I have two requests related to this:

1. Please separate addition of branding directives to add colors/fonts etc. and any changes to other aspects of the UI into two separate PRs.
2. Please sync up with me once you've had a chance to talk to Arti about what you'd like to do.

sean-roberts commented:
Since last review: 

- Removed the border/margin changes to be discussed another day
- Fixed app background height issue
- Added the drop-down arrow on the post button to reflect the ctaTextColor
